tasks.php

```diff
@@ -1470,24 +1499,45 @@ function TaskComments($tid)
     $result = DPDatabase::query($sql);
 
     echo "<h2>" . _("Comments") . "</h2>";
+    $Parsedown = new Parsedown();
+    $Parsedown->setSafeMode(true);
     while ($row = mysqli_fetch_assoc($result)) {
-        echo "<div class='task-comment'>";
+        $comment_id = create_anchor_for_comment($row['u_id'], $row['comment_date']);
+        $is_owner = $requester_u_id == $row['u_id'];
```

Comment:
As part of allowing SAs to edit comments, consider changing `$is_owner` to `$can_edit_comment` and then:
```
$can_edit_comment = $requester_u_id == $row['u_id'] || user_is_a_sitemanager();
```